content/docs/design/amp-html-layout/layouts_demonstrated.md

```diff
@@ -0,0 +1,138 @@
+---
+$title: Demonstrating AMP layouts
+$order: 5
+components:
+    - selector
+---
+
+This page demonstrates the different [layouts in AMP](https://www.ampproject.org/docs/design/amp-html-layout#layout), and how they work when a screen is resized.
+ 
+<div class="main-layouts">
+
+  <!-- fill -->
+  <div class="layout-item">
+    <h3>fill</h3>
+     <div class="description">With a <code>fill</code> layout, the element takes the space available to it—both width and height. In other words, the layout and size of a fill element matches its parent.</div>
+    <div class="fixed-container">
+      <div class="parent">
+        <div class="wrapper">
+            <amp-img layout="fill" src="/static/img/docs/layouts/blue.jpg"></amp-img>
+        </div>
+      </div>
+    </div>
+  </div>
+
+ <!-- fixed -->
+      <div class="layout-item">
+        <h3>fixed</h3> 
+        <div class="description">With a <code>fixed</code> layout, the element retains its fixed dimensions based on the element's width and height attributes (no responsiveness).</div>
+        <div class="fixed-container">
+          <div class="parent">
+            <div class="wrapper">
+              <amp-img layout="fixed"
+                src="/static/img/docs/layouts/blue.jpg"
+                width="200" height="150">
+              </amp-img>
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <!-- fixed-height -->
+      <div class="layout-item">
+        <h3>fixed-height</h3>
+        <div class="description">With a <code>fixed-height</code> layout, the element takes the space available to it but keeps the height unchanged. This layout works well for elements that need to position content horizontally (e.g., <code>amp-carousel</code>).</div>
+        <div class="fixed-container">
+          <div class="parent">
+            <div class="wrapper">
+              <amp-img layout="fixed-height"
+                src="/static/img/docs/layouts/blue.jpg" height="150">
+              </amp-img>
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <!-- flex-item -->
+      <div class="layout-item">
+        <h3>flex-item</h3> 
+        <div class="description">With a <code>fixed-height</code> layout, the element and other elements in its parent take the parent container's remaining space when the parent is a flexible container (i.e., display: <code>flex</code>).</div>
```

Comment:
s/fixed-height/flex-item

I don't quite understand the flex-item behaviour from the diagram and the explanation. I always thought it just makes the element behave as defined by the flex container (which usually means resizing all elements together).

Reply:
Thanks for the comments.

Based on the AMP's team's own [sample](https://github.com/ampproject/amphtml/blob/master/examples/layout-flex-item.amp.html), it doesn't look like the child items resize to "fit" the container.  Am I misunderstanding?

Reply:
You're right, it's just what I mostly used flex layout for. 

Suggestion: "With a <code>flex-item</code> layout, the element will take the available space based on the flexbox model."

via https://github.com/ampproject/amphtml/issues/2778

Reply:
Good wording! I've rewritten.